.github/workflows/push_tests.yml

```diff
@@ -3,7 +3,7 @@ name: Push Tests
 on:
   push:
     branches:
-      - main
+      - "**"
```

Comment:
Is this OK? I think it's useful to have WIP branches automatically tested even if they're not yet a PR, but LMK if there's some history here

Reply:
I can't think of any reason it was set to main other than not wanting to fill up the GHA queue with a ton of tests unrelated to active PRs. I think changing to all branches is fine for now, and if there's pushback we can revert.

Reply:
<img width="818" alt="image" src="https://user-images.githubusercontent.com/16548075/189679692-d56205e5-f39f-4666-9ae3-c8c668df60bd.png">
^ 10 checks now